mlc_config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ignorePatterns": [
+    { "pattern": "^/" },
+    { "pattern": "^./" },
+    { "pattern": "^../" }
+  ],
+  "timeout": "20s"
+}
```

Comment:
this is the configuration for the [markdown link check](https://github.com/tcort/markdown-link-check) program.

- the timeout is 20s because we have a few archive.org links which can be erroneously marked as dead because they take a while to respond
- the `ignorePatterns` are set to `/`, `./` and `../` because that lets us have the markdown link checker ignore all the _internal_ links, the checking of which is left to Docusaurus (it does a pretty good job already)

One consequence of that `ignorePatterns` bit that you'll notice is that it requires us to always use `./` or `../` for internal links in order to ensure that the link checker doesn't report spuriously broken links.

basically, if it encounters

```md
link to [some page](dir/page)
```

it will try to resolve `dir/page` as a URL, since it doesn't understand Docusaurus' format and algorithm for turning that string into a link to, say, `/dir/page`. 

So we just need to use the `./dir/page` path instead, which is, as far as I can tell, equally supported by Docusaurus.

Reply:
I love it - I hate this config uses JSON (not that it's our choice) and we can't add this as comments :-(